Cargo.toml

```diff
@@ -21,6 +21,7 @@ ansi-parsing = []
 libc = "0.2.30"
 unicode-width = { version = "0.1", optional = true }
 lazy_static = "1.4.0"
+regex = "1.4.2"
```

Comment:
This should be an optional dependency. Code that depends on it then should also be feature gated.